.github/workflows/ci.yml

```
@@ -40,14 +40,19 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Install build
+        run: python -Im pip install build -c test-requirements.txt
```

This would leak into the underlying PEP 517 ephemeral build envs.
```suggestion
        env:
          PIP_CONSTRAINT: test-requirements.txt
        run: python -Im pip install build
```